File: Java/Problem_028.java
public class Main {
    public static void main(String[] args) {
        final int N = 1001;
        long total = 1;

        for (long n = 3; n <= N; n += 2) {
            total += 4 * n * n - 6 * (n - 1);
        }

        System.out.println(total); // 669171001
    }
}

File: Python/Problem_028.py
def solve(N: int = 1001) -> int:
    total = 1
    for n in range(3, N + 1, 2):
        total += 4 * n * n - 6 * (n - 1)
    return total

if __name__ == "__main__":
    print(solve())  # 669171001
